distributedlog-core/src/main/java/org/apache/distributedlog/tools/DistributedLogTool.java

```diff
@@ -1665,7 +1665,7 @@ protected void repairLogSegment(BookKeeperAdmin bkAdmin,
                 System.out.println("Skip inprogress log segment " + segment);
                 return;
             }
-            LedgerHandle lh = bkAdmin.openLedger(segment.getLogSegmentId(), true);
```

Comment:
Thanks. This is the difference of bookkeeper apache version and twitter version.  Apache version does not have the second arg, so need to change all the caller of it in DL.